Firmware/Marlin_main.cpp

```diff
@@ -6120,7 +6120,7 @@ if(eSoundMode!=e_SOUND_MODE_SILENT)
         int8_t pin_number = LED_PIN;
         if (code_seen('P'))
           pin_number = code_value();
-        for(int8_t i = 0; i < (int8_t)(sizeof(sensitive_pins)/sizeof(int8_t)); i++)
+        for(int8_t i = 0; i < (int8_t)(sizeof(sensitive_pins)/sizeof(*sensitive_pins)); i++)
```

Comment:
please use
`sizeof(sensitive_pins)/sizeof(sensitive_pins[0])`

I know it has the same effect, but its intent is much more obvious

Reply:
done